stylesheets/partials/units/_button.scss

```diff
@@ -43,6 +43,13 @@ button {
     padding: .5em 1em;
 }
 
+.btn-default,
+.btn-primary,
+.buttonPrimary,
+.seButton {
```

Comment:
Are any other button types in use?  You might want to cover all the Bootstrap button classes to be safe.

Reply:
I believe there are other bootstrap buttons that do not require this style.